build.sbt

```diff
@@ -361,16 +381,8 @@ lazy val commonSettings = Seq(
   },
   scalaVersion := "2.10.5",
   crossScalaVersions := Seq(scalaVersion.value, "2.11.7"),
-  jvmTarget := "1.7",
-  scalacOptions ++= Seq(
-    "-deprecation",
-    "-feature",
-    "-language:implicitConversions",
-    "-language:higherKinds",
-    s"-target:jvm-${jvmTarget.value}",
-    "-unchecked",
-    "-Xlint"
-  ),
+  jvmTarget <<= scalaVersion.map { (v: String) => if (v.startsWith("2.10")) "1.7" else "1.8" },
```

Comment:
I think when we make the plunge to Java 1.8, we should go even for Scala 2.10.  Java 8 time classes on the model, Jetty 9.3, and HTTP 2 support are things that Scala 2.10 can benefit from.